### ISSUE TITLE
rule.struct-tag false positive on `validate:omitempty`

### ISSUE THREAD

**Issue by brianrietta** [mgechev/revive#1524]
**Describe the bug**
revive does not seem to recognize that the 'omitempty' tag is valid in 'validate' tags

**To Reproduce**
1. I updated revive `go install github.com/mgechev/revive@latest`
2. I run it with the following flags & configuration file: 

```shell
# flags

revive -formatter=stylish -config=revive.toml ./...`
```

```toml
# config file
ignoreGeneratedHeader = false
severity = "error"
confidence = 0.8
errorCode = 0
warningCode = 0

[directive.specify-disable-reason]
[rule.add-constant]
    arguments = [{maxLitCount = "3",allowStrs ="\"\",\"sfn.DescribeExecution: %w\",",allowInts="0,1,2,3",allowFloats="0.0"}]
[rule.argument-limit]
    arguments =[5]
[rule.atomic]
[rule.blank-imports]
[rule.bool-literal-in-expr]
[rule.call-to-gc]
[rule.confusing-naming]
    severity = "warning"
[rule.confusing-results]
    severity = "warning"
[rule.constant-logical-expr]
[rule.context-as-argument]
[rule.context-keys-type]
[rule.cyclomatic]
    arguments =[9]
    severity = "warning"
[rule.deep-exit]
[rule.duplicated-imports]
[rule.empty-block]
    severity = "warning"
[rule.empty-lines]
[rule.error-naming]
[rule.error-return]
[rule.error-strings]
[rule.errorf]
[rule.exported]
[rule.flag-parameter]
[rule.function-result-limit]
    arguments = [3]
[rule.get-return]
[rule.if-return]
[rule.increment-decrement]
[rule.indent-error-flow]
[rule.imports-blacklist]
    arguments =["http://github.com/pkg/errors"]
    severity = "warning"
[rule.import-shadowing]
[rule.modifies-value-receiver]
[rule.package-comments]
    severity = "warning"
[rule.range]
[rule.range-val-in-closure]
[rule.receiver-naming]
[rule.redefines-builtin-id]
[rule.struct-tag]
  arguments = ["json,inline"]
[rule.superfluous-else]
[rule.time-naming]
[rule.var-naming]
    arguments = [["ID"]]
[rule.var-declaration]
[rule.unexported-return]
[rule.unhandled-error]
    arguments = ["fmt.Println","fmt.Printf"]
[rule.unnecessary-stmt]
[rule.unreachable-code]
[rule.unused-parameter]
[rule.unused-receiver]
[rule.waitgroup-by-value]
```
I have tried adding `validate` to the arguments under rule.struct-tag but got the same undesired result

** Actual Behavior **
revive complains about the presence of the `omitempty` value in `validate` tags everywhere it appears, and this did not happen before

**Expected behavior**
using `omitempty` in combination with other validate tags is valid, but on its own revive should probably question it unless the type of the property is a pointer to a custom struct that has its own validate tags

**Logs**

**Desktop (please complete the following information):**
 - OS: [Mac Tahoe 26]
 - Version of Go 1.25.1

**Additional context**

**Comment by chavacava** [comment]
Hi @brianrietta, thanks for reporting the issue.

**Comment by chavacava** [comment]
@brianrietta the PR #1525 should fix the problem.

Side note: the configuration to accept `omitempty` in `validate` tags is:

```toml
[rule.struct-tag]
arguments=["validate,omitempty"]
```

**Comment by brianrietta** [comment]
hey, thank you so much for the quick turnaround! I can confirm on the latest version of revive and using the suggested config, revive no longer yells at me for valid validator/v10 tags
